libse/Forms/FixCommonErrors/FixMusicNotation.cs

```diff
@@ -17,11 +17,30 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
                 {
                     var oldText = p.Text;
                     var newText = oldText;
-
+                    bool containsFontTag = oldText.Contains("<font ", StringComparison.OrdinalIgnoreCase);
                     foreach (string musicSymbol in musicSymbols)
                     {
-                        newText = newText.Replace(musicSymbol, Configuration.Settings.Tools.MusicSymbol);
-                        newText = newText.Replace(musicSymbol.ToUpper(), Configuration.Settings.Tools.MusicSymbol);
+                        if (containsFontTag && musicSymbol == "#")
+                        {
+                            var idx = newText.IndexOf('#');
+                            while (idx >= 0)
+                            {
+                                // <font color="#808080">NATIVE HAWAIIAN CHANTING</font>
+                                var isInsideFontTag = (idx < 13) ? true : (newText[idx - 1] == '"' && (newText.Length > idx + 2 && char.IsDigit(newText[idx + 1]) && char.IsDigit(newText[idx + 2])));
```

Comment:
Also, I think, `#` is followed by hexadecimal digits:
 `Uri.IsHexDigit(newText[idx + 1]) && Uri.IsHexDigit(newText[idx + 2])`